bin/reth/src/node/mod.rs

```diff
@@ -759,15 +794,15 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
         // try to look up the header in the database
         if let Some(header) = header {
             info!(target: "reth::cli", ?tip, "Successfully looked up tip block in the database");
-            return Ok(header.seal_slow())
+            return Ok(header.seal_slow());
```

Comment:
```suggestion
            return Ok(header.seal_slow())
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145